solr/core/src/test/org/apache/solr/handler/admin/HealthCheckHandlerTest.java

```diff
@@ -112,36 +112,6 @@ public void testHealthCheckHandler() throws Exception {
       newJetty.stop();
     }
 
-    // add a new node for the purpose of negative testing
-    // negative check that if core container is not available at the node
-    newJetty = cluster.startJettySolrRunner();
-    try (SolrClient solrClient = getHttpSolrClient(newJetty.getBaseUrl().toString())) {
-
-      // positive check that our (new) "healthy" node works with direct http client
-      assertEquals(CommonParams.OK, req.process(solrClient).getResponse().get(CommonParams.STATUS));
-
-      // shutdown the core container of new node
-      newJetty.getCoreContainer().shutdown();
```

Comment:
I considered this portion of the test invalid since it quite intentionally (see the opening comment) shuts down the container without touching Jetty.  That's simply not how Jetty shuts down; only Jetty's lifecycle initiates an orderly shutdown.  I could test trying an attempt to do a request against a non-existent server but that seemed silly.